README.md

```diff
@@ -109,51 +194,67 @@ default ✓ [======================================] 10 VUs  1m0s
-  - **`incr(key: String) -> Promise<Number>`**: increments the number stored at `key` by one. If the key does not exist, it is set to zero before performing the operation. An error is returned if the key contains a value of the wrong type, or contains a string that cannot be represented as an integer. Returns the value of key after the increment.
-  - **`incrby(key: String, increment: Number) -> Promise<Number>`**: increments the number stored at `key` by `increment`. If the key does not exist, it is set to zero before performing the operation. An error is returned if the key contains a value of the wrong type, or contains a string that cannot be represented as an integer.
-  - **`decr(key: String) -> Promise<Number>`**: decrements the number stored at `key` by one. If the key does not exist, it is set to zero before performing the operation. An error is returned if the key contains a value of the wrong type, or contains a string that cannot be represented as an integer.
-  - **`decrby(key: String, decrement: Number) -> Promise<Number>`**: decrements the number stored at `key` by `decrement`. If the key does not exist, it is set to zero before performing the operation. An error is returned if the key contains a value of the wrong type, or contains a string that cannot be represented as an integer.
-  - **`expire(key: String, seconds: Number) -> Promise<Boolean>`**: sets a timeout on key, after which the key will automatically be deleted. Note that calling Expire with a non-positive timeout will result in the key being deleted rather than expired.
-  - **`ttl(key: String) -> Promise<Number>`**: returns the remaining time to live of a key that has a timeout.
-  - **`persist(key: String) -> Promise<Boolean>`**: removes the existing timeout on key.
-  - **`random() -> String`**: returns a random key.
+
+| Redis Command | Module function signature                                             | Description                                                                                                                                                                                                           | Returns                                                                                                                                                                                                                                     |
+| :------------ | :-------------------------------------------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```

Comment:
```suggestion
| Redis Command | Module function signature | Description | Returns |
| ---------------------- | ----------------------------------- | --------------- | ---------- |
```

Reply:
Good catch! You have laser eyes, I have to spread vscode over my two screens to see the tables in full 😄